allennlp/training/trainer.py

```diff
@@ -355,7 +348,7 @@ def __init__(
         local_rank: int = 0,
         world_size: int = 1,
         num_gradient_accumulation_steps: int = 1,
-        opt_level: Optional[str] = None,
+        use_amp: bool = False,
```

Comment:
Better name for this parameter? Maybe just `amp`?

Reply:
`use_amp` seems fine to me; I'd prefer `use_amp` to just `amp`.